app/views/user_annotations/edit.html.erb

```diff
@@ -1,6 +1,8 @@
-<h1>Editing '<em><%= @user_annotation.name %></em>'</h1>
-<em><h4><%= @user_annotation.subsampled_at %></h4></em>
```

Comment:
This edit permission check is unnecessary as you're checking edit permission both in the index view (to render the button) and in a before_filter to redirect if the user doesn't have permission.